src/CMakeLists.txt

```diff
@@ -58,15 +58,9 @@ if(GENERATE_POSITION_INDEPENDENT_CODE OR BINDINGS)
   set_target_properties(${PROJECT_NAME}_lib PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
 endif()
 
-# add nlohmann::json library
-set(JSON_BuildTests # cmake-lint: disable=C0103
-    OFF
-    CACHE INTERNAL "")
-set(JSON_MultipleHeaders # cmake-lint: disable=C0103
-    OFF
-    CACHE INTERNAL "")
-add_subdirectory("${PROJECT_SOURCE_DIR}/extern/json" "extern/json" EXCLUDE_FROM_ALL)
 target_link_libraries(${PROJECT_NAME}_lib PUBLIC nlohmann_json)
+# add MQT::QFR library
+add_subdirectory("${PROJECT_SOURCE_DIR}/extern/qfr" "extern/qfr" EXCLUDE_FROM_ALL)
```

Comment:
Move these lines to the top of the file to have everything available.

Reply:
done